File: aurora-base/admin-common/src/main/java/xyz/xcye/admin/dto/RolePermissionDTO.java
package xyz.xcye.admin.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author qsyyke
 * @date Created in 2022/5/7 19:26
 */

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RolePermissionDTO {
    /**
     * 角色名称
     */
    private String roleName;

    /**
     * 用户名
     */
    private String username;

    /**
     * 角色的状态
     */
    private Boolean roleStatus;

    /**
     * 权限名称
     */
    private String permissionName;

    /**
     * 此角色可以允许访问的路径
     */
    private String path;

    /**
     * 用户的userUid
     */
    private Long userUid;
}

File: auroraAdminWeb/src/bean/dto/RolePermissionDto.ts
interface RolePermissionDto {
	/**
	 * 角色名称
	 */
	roleName?: string | null;

	username?: string | null;

	/**
	 * 角色的状态
	 */
	roleStatus?: boolean | null;

	/**
	 * 权限名称
	 */
	permissionName?: string | null;

	/**
	 * 此角色可以允许访问的路径
	 */
	path?: string | null;

	/**
	 * 用户的userUid
	 */
	userUid?: string | null;
}
